exercises/concept/meltdown-mitigation/.meta/exemplar.py

```diff
@@ -67,8 +67,9 @@ def fail_safe(temperature, neutrons_produced_per_second, threshold):
     - `temperature * neutron per second` is not in the above-stated ranges ==  'DANGER'
     """
     output = temperature * neutrons_produced_per_second
-    operational_percentage = int((output / threshold) * 100)
+    operational_percentage = (output / threshold) * 100
     safety_range = 'UNKNOWN'
+    print(operational_percentage)
```

Comment:
Generally non-CLI code should not contain any `print()` statements.

Reply:
This was a debugging leftover... Thanks,